app/containers/RoomItem/useRowHeight.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { createContext, useContext } from 'react';
+import { PixelRatio } from 'react-native';
+
+interface IRowHeightProvider {
+	children: React.ReactNode;
```

Comment:
We shouldn't use `ReactNode`, since it makes it possible to return plain values, like `'test'` and that would crash the app.
You should do `ReactElement | null` (if it can be null)